vnext/Directory.Build.props

```diff
@@ -13,12 +13,16 @@
     <!-- This property sets the CscTask to print the full path of the csharp file on errors and warnings rather than just the filename. -->
     <GenerateFullPaths>true</GenerateFullPaths>
 
+    <!-- This property controls whether the React Native inspector (Hermes direct debugger) gets built  -->
+    <EnableHermesInspector>false</EnableHermesInspector>
```

Comment:
do we need this as a separate property or can we just use `USE_HERMES` instead?

Reply:
HI @asklar  .. Sorry for a super late response.. I've a slightly better solution.. but unfortunately this PR is closed. I think most of your feedbacks are alrady taken care in the new PR.  Including this one.